### PR TITLE
[feat/aut873] - Enhancing FilePicker entries to be displayed in human readable format

### PR DESCRIPTION
**Pull request by ghost**
Objectives

- Changing File sizes from bytes to the equivalent in KB and GB whenever appropriate.
- Changing Last Modified Date to a human readable format.

### CODE REVIEW

File: src/smartpeak/source/core/Utilities.cpp [hunk 1]
```diff
@@ -705,4 +705,82 @@ namespace SmartPeak
...
+    current_time = (std::time_t)std::atoll(date.c_str());
+    strptime(date.c_str(), "%F %T", &entry_date_tm);
+    time_t entry_date_time = mktime(&entry_date_tm);
+    
+    if (entry_date_tm.tm_mday == current_time_tm->tm_mday
+        && entry_date_tm.tm_mon == current_time_tm->tm_mon
+        && entry_date_tm.tm_year == current_time_tm->tm_year)
+    {
+      std::strftime(date_time_buffer, sizeof date_time_buffer, "%R", std::localtime(&entry_date_time));
+      directory_entry.entry_contents[3] = "Today at " + std::string(date_time_buffer);
```
Comment (ChristopherAbram) [inline src/smartpeak/source/core/Utilities.cpp:782]: Nice part to inform user that it happened today, not just plain date 👍

**Review by ChristopherAbram — APPROVED** [review]
Good to go!

File: src/smartpeak/source/core/Utilities.cpp (hunk 1 above)
Comment (ghost) [reply src/smartpeak/source/core/Utilities.cpp:782]: > Good to go!

yes, however it has some issues with Daylight saving and so it's displaying last modified dates with an hour earlier.

**Review by bertrandboudaud — APPROVED** [review]
looks good!

**Pull request merged**